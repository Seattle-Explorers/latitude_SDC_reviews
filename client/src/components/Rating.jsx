import React from 'react';

const Rating = () => {
  return(
    <div>
      KMS
    </div>
  )
}

export default Rating;