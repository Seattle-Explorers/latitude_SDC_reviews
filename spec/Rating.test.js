import 'jsdom-global/register';
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-enzyme';
import 'jest-styled-components';

import mockData from '../__mocks__/mockData';
import Rating from '../client/src/components/PreviewRating';

describe('Preview Rating Components', () => {
  const { reviews, ...rest } = mockData;
  it('should render', () => {
    const component = <Rating overview={rest} />;

    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
