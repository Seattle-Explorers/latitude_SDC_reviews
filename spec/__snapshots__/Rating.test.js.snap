// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview Rating Components should render 1`] = `
.c0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-right: 50px;
  padding-left: 20px;
  width: 35%;
}

.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 2em;
  font-weight: bolder;
}

.c2 {
  margin-left: 10px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-top: 16px;
  font-size: 15px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  width: 75%;
}

.c4 {
  width: 25%;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c8 {
  font-size: 13px;
  font-weight: bolder;
}

.c6 {
  margin-top: 5px;
  width: 33%;
  height: 0.25em;
  float: left;
  border-radius: 15%;
  background-color: #B3B3B3;
  margin-right: 5px;
}

.c7 {
  margin-top: 0;
  width: 20%;
  height: 100%;
  float: left;
  border-radius: 15%;
  background-color: black;
  margin-right: 5px;
}

.c9 {
  margin-top: 0;
  width: 24%;
  height: 100%;
  float: left;
  border-radius: 15%;
  background-color: black;
  margin-right: 5px;
}

.c10 {
  margin-top: 0;
  width: 52%;
  height: 100%;
  float: left;
  border-radius: 15%;
  background-color: black;
  margin-right: 5px;
}

.c11 {
  margin-top: 0;
  width: 80%;
  height: 100%;
  float: left;
  border-radius: 15%;
  background-color: black;
  margin-right: 5px;
}

.c12 {
  margin-top: 0;
  width: 38%;
  height: 100%;
  float: left;
  border-radius: 15%;
  background-color: black;
  margin-right: 5px;
}

.c13 {
  margin-top: 0;
  width: 32%;
  height: 100%;
  float: left;
  border-radius: 15%;
  background-color: black;
  margin-right: 5px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <svg
      color="#FF585D"
      fill="currentColor"
      height="1em"
      size="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": "#FF585D",
        }
      }
      viewBox="0 0 576 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
      />
    </svg>
    <span
      className="c2"
    >
      2.05
       (
      7
       reviews)
    </span>
  </div>
  <div
    className="c3"
  >
    <div
      className="c4"
    >
      Cleaniness
    </div>
    <div
      className="c5"
    >
      <div
        className="c6"
      >
        <div
          className="c7"
          color="black"
          height="100%"
          width="20%"
        />
      </div>
      <span
        className="c8"
      >
        1
      </span>
    </div>
  </div>
  <div
    className="c3"
  >
    <div
      className="c4"
    >
      Accuracy
    </div>
    <div
      className="c5"
    >
      <div
        className="c6"
      >
        <div
          className="c9"
          color="black"
          height="100%"
          width="24%"
        />
      </div>
      <span
        className="c8"
      >
        1.2
      </span>
    </div>
  </div>
  <div
    className="c3"
  >
    <div
      className="c4"
    >
      Communication
    </div>
    <div
      className="c5"
    >
      <div
        className="c6"
      >
        <div
          className="c10"
          color="black"
          height="100%"
          width="52%"
        />
      </div>
      <span
        className="c8"
      >
        2.6
      </span>
    </div>
  </div>
  <div
    className="c3"
  >
    <div
      className="c4"
    >
      Location
    </div>
    <div
      className="c5"
    >
      <div
        className="c6"
      >
        <div
          className="c11"
          color="black"
          height="100%"
          width="80%"
        />
      </div>
      <span
        className="c8"
      >
        4
      </span>
    </div>
  </div>
  <div
    className="c3"
  >
    <div
      className="c4"
    >
      Check-in
    </div>
    <div
      className="c5"
    >
      <div
        className="c6"
      >
        <div
          className="c12"
          color="black"
          height="100%"
          width="38%"
        />
      </div>
      <span
        className="c8"
      >
        1.9
      </span>
    </div>
  </div>
  <div
    className="c3"
  >
    <div
      className="c4"
    >
      Value
    </div>
    <div
      className="c5"
    >
      <div
        className="c6"
      >
        <div
          className="c13"
          color="black"
          height="100%"
          width="32%"
        />
      </div>
      <span
        className="c8"
      >
        1.6
      </span>
    </div>
  </div>
</div>
`;
